tests/Hooks/Migration/Unit/MigratesHookTest.php

```diff
@@ -29,6 +29,7 @@ class MigratesHookTest extends TestCase
 
     protected function afterSetUp()
     {
+        resolve('migrator')->path(__DIR__);
```

Comment:
Also check the other comment regarding the `afterSetUp` function

Reply:
Same as my other comment.